backend/src/workspaces/workspaces.controller.ts

```diff
@@ -68,7 +68,7 @@ export class WorkspacesController {
 		@Req() req: AuthroizedRequest,
 		@Param("workspace_slug") workspaceSlug: string
 	): Promise<FindWorkspaceResponse> {
-		return this.workspacesService.findOneBySlug(req.user.id, workspaceSlug);
+		return this.workspacesService.findOneBySlug(req.user.id, encodeURIComponent(workspaceSlug));
```

Comment:
I think we don't have to call `encodeURIComponent` here.
Because the parameter is already slug.

Reply:
Oh, I see.
Then, I think it's okay.